manifests/plugin/remote_execution/ssh/params.pp

```diff
@@ -0,0 +1,10 @@
+# Remote Execution SSH default parameters
+class foreman_proxy::plugin::remote_execution::ssh::params {
+  $enabled           = true
+  $listen_on         = 'https'
+  $generate_keys     = true
+  $ssh_identity_dir  = "${::foreman_proxy::dir}/.ssh"
```

Comment:
I can remove this option entirely, and let ssh-keygen do the deafult thing which will be to put it in foreman_proxy home directory, but that will require smart_proxy_remote_execution_ssh changes to never allow customization of the path.  That's a big limitation on us.  Is there nothing else we can do?

@iNecas @domcleal any ideas?